src/ubkg_api/wsgi.py

```diff
@@ -0,0 +1,7 @@
+from app import UbkgAPI
+
+application = UbkgAPI('./app.cfg').app
```

Comment:
Since we are using the `src/ubkg_api/instance/app.cfg` and it's being git ignored, no need to use a separate config file from the current directory, which is not git ignored.

Reply:
Discussed with Joe on Slack.  Only `app.cfg` needed is one in `instance`.  But  removed initial `./`, as this was not necessary in my development environment.